hrms/overrides/custom_employee_advance.py

```diff
@@ -275,6 +276,11 @@ def validate_approver(self):
         ):
             frappe.throw(_(f"Approval selected does not have L1 or L2 Expense Approver Role"))
 
+    def calulate_inr_amount(self):
+        get_default_company_currency = frappe.db.get_value("Company", CompanyConstants.SUITE42, "default_currency")
+        if self.currency != get_default_company_currency:
+            self.advance_amt_inr = flt(self.exchange_rate*self.advance_amount, 1)
```

Comment:
make exchange rate readonly and ask for amount in user currency and amount in company currency

Reply:
Done.